builds/misc/templates/build-api-proxy.yaml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  name: 'API proxy'
+  imageName: 'api-proxy-module'
+  namespace: 'microsoft'
+  project: 'api-proxy-module'
+  configuration: 'release'
```

Comment:
Should we build "debug" when calling from images.yaml?

Reply:
No, everything else is using release, we should keep it consistent.

Reply:
Be specific, you are referring to what we did in Edgelet.yaml. I think we should clean up if we doesn't pass in different value for configuration, but it is not a big problem.